docs/pages/reference/resources.mdx

```diff
@@ -222,3 +226,97 @@ Global configuration options for the Web UI served by the Proxy Service. This re
+  second_factor: "otp"
+
+  # The name of the OIDC or SAML connector. if this is not set, the first connector in the backend is used.
+  connector_name: ""
+
+  # webauthn is the settings for server-side Web authentication support.
+  webauthn:
+    # rp_id is the ID of the Relying Party.
+    # It should be set to the domain name of the Teleport installation.
+    #
+    # IMPORTANT: rp_id must never change in the lifetime of the cluster, because
+    # it's recorded in the registration data on the WebAuthn device. If the
+    # ri_id changes, all existing WebAuthn key registrations will become invalid
+    # and all users who use WebAuthn as the second factor will need to
+    # re-register.
+    rp_id: teleport.example.com
+    # Allow list of device attestation CAs in PEM format.
+    # If present, only devices whose attestation certificates match the
+    # certificates specified here may be registered (existing registrations are
+    # unchanged).
+    # If supplied in conjunction with `attestation_denied_cas`, then both
+    # conditions need to be true for registration to be allowed (the device
+    # MUST match an allowed CA and MUST NOT match a denied CA).
+    # By default all devices are allowed.
+    attestation_allowed_cas: []
+    # Deny list of device attestation CAs in PEM format.
+    # If present, only devices whose attestation certificates don't match the
+    # certificates specified here may be registered (existing registrations are
+    # unchanged).
+    attestation_denied_cas: []
+
+  # Enforce per-session MFA or PIV-hardware key restrictions on user login sessions.
+  # Possible values: true, false, "hardware_key", "hardware_key_touch"
+  require_mfa_type: "off"
+
+  # Sets whether connections with expired client certificates will be disconnected.
+  disconnect_expired_cert: false
+
+  # Sets whether headless authentication is allowed.
+  # Headless authentication requires WebAuthn.
+  # Defaults to true if webauthn is configured.
+  allow_headless: false
+
+  # Sets whether local auth is enabled alongside any other authentication
+  # type.
+  allow_local_auth: true
+
+  # Sets whether passwordless authentication is allowed.
+  # Requires Webauthn to work.
+  allow_passwordless: false
+
+  # Sets the message of the day for the cluster.
+  message_of_the_day: ""
+
+  # idp is a set of options related to accessing IdPs within Teleport. Requires Teleport Enterprise
+  idp:
+    # options related to the Teleport SAML IdP.
+    saml:
+      # enables access to the Teleport SAML IdP.
+      enabled: true
+
+  # locking_mode is the cluster-wide locking mode default.
+  # Possible values: "strict" or "best_effort"
+  locking_mode: best_effort
+
+  # default_session_ttl defines the default TTL (time to live) of certificates
+  # issued to the users on this cluster.
+  default_session_ttl: "12h"
+
+  type: local
+
+  # Deprecated by Webauthn: U2F are settings for the u2f device.
```

Comment:
Personally, I'd just omit this. We don't want any new users configuring U2F today.